crates/cheatcodes/src/utils.rs

```diff
@@ -219,3 +236,49 @@ fn derive_key<W: Wordlist>(mnemonic: &str, path: &str, index: u32) -> Result {
     let private_key = U256::from_be_bytes(wallet.signer().to_bytes().into());
     Ok(private_key.abi_encode())
 }
+
+#[cfg(test)]
+mod tests {
+    use alloy_primitives::FixedBytes;
+    use super::*;
+    use hex::FromHex;
+    use crate::CheatsConfig;
+    use std::{path::PathBuf, sync::Arc};
+
+    fn cheats() -> Cheatcodes {
+        let config = CheatsConfig {
+            ffi: true,
+            root: PathBuf::from(&env!("CARGO_MANIFEST_DIR")),
+            ..Default::default()
+        };
+        Cheatcodes { config: Arc::new(config), ..Default::default() }
+    }
+    
+    #[test]
+    fn test_sign_p256() {
+        let pk = "0xA8568B74282DCC66FF70F10B4CE5CC7B391282F5381BBB4F4D8DD96974B16E6B".parse().unwrap();
+        let digest = FixedBytes::from_hex("0x54705ba3baafdbdfba8c5f9a70f7a89bee98d906b53e31074da7baecdc0da9ad").unwrap();
```

Comment:
Where are these + solidity test vectors from, are they arbitrary or are there standard test vectors used that we should use?

Reply:
Was going to comment, these are from a test I did manually. There are some vectors here but they include a nonce https://github.com/RustCrypto/elliptic-curves/blob/master/p256/src/test_vectors/ecdsa.rs. 

We could leverage p256::ecdsa::VerifyingKey in these tests if you think that would be better.

Reply:
I'm mostly indifferent to where they are from as long as there's a way to verify the test vectors are actually correct, so using the ones from p256::ecdsa::VerifyingKey sgtm :)

Reply:
Well VerifyingKey would just be replying on this same library to verify whatever we put, so maybe we want to find something better https://docs.rs/p256/latest/p256/ecdsa/index.html

Reply:
That seems ok, since we want to verify our implementation of the library is correct when translated into our own rust methods and solidity cheats

Reply:
ack, updated tests